README.md

```diff
@@ -51,6 +51,29 @@ For more information on how to contribute, please see our [Contribution Guide](d
 1. Clone the repository (including submodules)
     - `git clone --recurse-submodules https://github.com/mit-dci/opencbdc-tx`
 
+# Build
+
+Use these directions if you want to build the source on your machine.
+If you just want to run the system, see "Run the Code" below.
+
+## macOS
+Ensure your development environment is set correctly for clang:
```

Comment:
```suggestion
Ensure your development environment is set correctly for clang (to reset: sudo xcode-select --reset):
```
This tip might be useful for those using the full Xcode.app

Reply:
When mentioning code (or commands) in documentation, they should probably be enclosed in backticks for code formatting (`like so`).